README.md

```diff
@@ -1,4 +1,4 @@
-# th2 common library (Java) (5.1.1)
+# th2 common library (Java) (5.2.0)
```

Comment:
Do we really need to update version if we don't even have a first release?

Reply:
I think yes, because we made versioning document to follow changes in projects for development.